webapp/channels/src/components/admin_console/custom_terms_of_service_settings/custom_terms_of_service_settings.tsx

```diff
@@ -200,8 +199,13 @@ export default class CustomTermsOfServiceSettings extends AdminSettings<Props, S
                 <BooleanSetting
                     key={'customTermsOfServiceEnabled'}
                     id={'SupportSettings.CustomTermsOfServiceEnabled'}
-                    label={<FormattedMessage {...messages.enableTermsOfServiceTitle}/>}
-                    helpText={<FormattedMarkdownMessage {...messages.enableTermsOfServiceHelp}/>}
+                    label={<FormattedMessage {...messages.enableTermsOfServiceTitle} />}
+                    helpText={<FormattedMessage
+                        {...messages.enableTermsOfServiceHelp}
```

Comment:
`enableTermsOfServiceHelp` string has not been updated with the XML `<a/>`

Reply:
@M-ZubairAhmed Can you check it now?

Reply:
thanks for adding the translation also need to be edited in en.json as well

Reply:
@M-ZubairAhmed I updated it